packages/web-components/src/components/quote/__stories__/quote.stories.react.tsx

```diff
@@ -0,0 +1,94 @@
+import DDSQuoteSourceCopy from '@carbon/ibmdotcom-web-components/es/components-react/quote/quote-source-copy';
+import DDSQuoteSourceBottomCopy from '@carbon/ibmdotcom-web-components/es/components-react/quote/quote-source-bottom-copy';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+import { QUOTE_TYPES, QUOTE_COLOR_SCHEMES } from '../quote';
+import readme from './README.stories.react.mdx';
+
+const types = {
+  [`${QUOTE_TYPES.DEFAULT}`]: QUOTE_TYPES.DEFAULT,
+  [`${QUOTE_TYPES.SINGLE_CURVED}`]: QUOTE_TYPES.SINGLE_CURVED,
+  [`${QUOTE_TYPES.DOUBLE_ANGLE}`]: QUOTE_TYPES.DOUBLE_ANGLE,
+  [`${QUOTE_TYPES.SINGLE_ANGLE}`]: QUOTE_TYPES.SINGLE_ANGLE,
+  [`${QUOTE_TYPES.LOW_HIGH_REVERSED_DOUBLE_CURVED}`]: QUOTE_TYPES.LOW_HIGH_REVERSED_DOUBLE_CURVED,
+  [`${QUOTE_TYPES.CORNER_BRACKET}`]: QUOTE_TYPES.CORNER_BRACKET,
+};
+
+const colorSchemes = {
+  [`${QUOTE_COLOR_SCHEMES.REGULAR}`]: QUOTE_COLOR_SCHEMES.REGULAR,
+  [`${QUOTE_COLOR_SCHEMES.INVERSE}`]: QUOTE_COLOR_SCHEMES.INVERSE,
+};
+
+export const Default = ({ parameters }) => {
+  const { copy, quoteMark, sourceHeading, sourceCopy, sourceBottomCopy, colorScheme } = parameters?.props?.Quote ?? {};
+  return (
+    <DDSQuote color-scheme={colorScheme} mark-type={quoteMark}>
+      {copy}
+      <DDSQuoteSourceHeading>{sourceHeading}</DDSQuoteSourceHeading>
+      <DDSQuoteSourceCopy>{sourceCopy}</DDSQuoteSourceCopy>
+      <DDSQuoteSourceBottomCopy>{sourceBottomCopy}</DDSQuoteSourceBottomCopy>
+      <DDSLinkWithIcon slot="footer" href="https://example.com">
+        Link with icon <ArrowRight20 slot="icon"></ArrowRight20>
+      </DDSLinkWithIcon>
+    </DDSQuote>
+  );
+};
+
+Default.story = {
+  parameters: {
+    knobs: {
+      Quote: ({ groupId }) => ({
+        copy: textNullable('Quote (copy):', 'Bringing together the technology and expertise for a new way to create', groupId),
+        quoteMark: select('Quote Mark (markType):', types, types.doubleCurved, groupId),
+        sourceHeading: textNullable('Source heading (source-heading slot)', 'John Doe', groupId),
+        sourceCopy: textNullable('Source copy (source-copy slot)', 'Senior Vice President', groupId),
+        sourceBottomCopy: textNullable('Source bottom copy (source-bottom-copy slot)', 'IBM Cloud', groupId),
+        colorScheme: select('Color Scheme (color-scheme)', colorSchemes, colorSchemes.regular, groupId),
+      }),
+    },
+    propsSet: {
+      default: {
+        Quote: {
+          copy: 'Bringing together the technology and expertise for a new way to create',
+          quoteMark: 'double-curved',
+          sourceHeading: 'John Doe',
+          sourceCopy: 'Senior Vice President',
+          sourceButtonCopy: 'IBM Cloud',
+        },
+      },
+    },
+  },
+};
+
+export default {
+  title: 'Components/Quote',
+  decorators: [
+    story => (
+      <div className="bx--grid">
+        <div className="bx--row">
+          <div className="bx--col-lg-11">{story()}</div>
```

Comment:
```suggestion
          <div className="bx--col-lg-10">{story()}</div>
```

Just to stay consistent with `web-components` story.